### DISCUSSION TITLE
#include errors detected. #1696.

### DISCUSSION THREAD

**Discussion by acinine** [microsoft/vscode-cpptools#7739] [Q&A]
I am starting to learn c++ with Microsoft Visual Studio Code.
I have written this:

#include <iostream>
int main(){
    std::cout << "Hello World!";
    std::cin.ignore;
    return 0;
}

For some reason, (#include) causes problems.
This was written when reviewing it:

include errors detected. Please update your includePath. Squiggles are disabled for this translation unit (C:\Users\(my_username)\Desktop\C++\python.cpp).
C/C++(1696)
cannot open source file "iostream"C/C++(1696)

**Comment by michelleangela** [comment]
@c0smo225 please see response at your posted issue https://github.com/microsoft/vscode-cpptools/issues/7740
